src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj

```diff
@@ -99,6 +99,7 @@
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
       <LanguageStandard>stdcpp17</LanguageStandard>
       <ForcedIncludeFiles>stdafx.h</ForcedIncludeFiles>
+      <ControlFlowGuard>Guard</ControlFlowGuard>
```

Comment:
Recommend reverting all changes to `*.vcxproj` files and adding this property to eng\targets\Cpp.Common.props

Reply:
@dougbu as I mentioned:
> I did the changes through the VS properties pages, so blame it for not simplifying these settings XD

I understand that it isn't pretty, but any time any property is changed through VS, it will create this ugly output repeated 4 times. Personally, I'd rather just rely on the tooling than having to always manually fix every property change.

Reply:
Why rely on the tooling for this when we almost never do for changes in `*.csproj` files? It's a one line change in one file when done manually -- no need to worry about mulitple platforms and configurations.